controllers/managedosversionchannel_controller.go

```diff
@@ -525,5 +530,16 @@ func filterChannelEvents() predicate.Funcs {
 			logger.V(log.DebugDepth).Info("Processing generic event", "Obj", e.Object.GetName())
 			return true
 		},
+		// Ignore pods creation
+		CreateFunc: func(e event.CreateEvent) bool {
```

Comment:
This is to prevent reconciling again immediately after creating the pod resource. we should only re-reconcile on pod status changes.

Reply:
well done, this was the extra reconcile loop we saw